packages/expo-video/ios/VideoView.swift

```diff
@@ -13,13 +13,42 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
   }
 
   var isFullscreen: Bool = false
+  var startPictureInPictureAutomatically = false {
+    didSet {
+      if #available(iOS 14.2, *) {
+        playerViewController.canStartPictureInPictureAutomaticallyFromInline = startPictureInPictureAutomatically
+      }
+    }
+  }
+
+  var allowPictureInPicture: Bool = false {
+    didSet {
+      if allowPictureInPicture {
+        let audioSession = AVAudioSession.sharedInstance()
+        do {
+          try audioSession.setCategory(.playback, mode: .default)
+          try audioSession.setActive(true)
```

Comment:
Shouldn't we set this when the player starts PiP and not when someone sets the prop? I can imagine some issues here, for example when the other module needs to set to another category.

Reply:
We need to set it when the prop changes, because the PiP button in the native controls shows up automatically only when a correct audioSession category is set. If we don't set it here the button will only appear after the user calls "startPictureInPicture". 
I'm not sure how to deal with the potential issues though :/